include/wx/osx/cocoa/private.h

```diff
@@ -203,14 +223,33 @@ public :
 
 protected:
     WXWidget m_osxView;
+    
+    // begins processing of native key down event, storing the native event for later wx event generation
+    void BeginNativeKeyDownEvent( NSEvent* event );
+    // done with the current native key down event
+    void EndNativeKeyDownEvent();
+    // allow executing text changes without triggering key down events
+
+    // is currently processing a native key down event
+    bool IsInNativeKeyDown();
```

Comment:
```suggestion
    bool IsInNativeKeyDown() const;
```
just because a non-const accessor like this looks suspicious and raises questions about why isn't it const